flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/job/JobCancellationHandler.java

```diff
@@ -83,7 +83,7 @@ public JobCancellationHandler(
 				terminationFuture = gateway.cancelJob(jobId, timeout);
 				break;
 			case STOP:
-				throw new RestHandlerException("The \"stop\" command has been removed. Please use \"stop-with-savepoint\" instead.", HttpResponseStatus.PERMANENT_REDIRECT);
+				throw new RestHandlerException("The \"stop\" command has been removed. Please use \"stop\" instead.", HttpResponseStatus.PERMANENT_REDIRECT);
```

Comment:
The exception message does not make much sense anymore, so it has to be updated.

Reply:
Updated.